src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/converter/TimestampingServiceConverter.java

```diff
@@ -57,7 +59,15 @@ public TimestampingService convert(String url)  {
         return timestampingService;
     }
 
-    public List<TimestampingService> convert(Iterable<String> urls)  {
-        return Streams.stream(urls).map(this::convert).collect(Collectors.toList());
+    public TimestampingService convert(TspType tsp)  {
+        TimestampingService timestampingService = new TimestampingService();
+        timestampingService.setUrl(tsp.getUrl());
+        timestampingService.setName(tsp.getName());
+        return timestampingService;
+    }
+
+    public List<TimestampingService> convert(Iterable<?> list)  {
+        return Streams.stream(list).map(n -> n instanceof String ? convert((String)n) : convert((TspType)n))
```

Comment:
Having methods `convert(Iterable<String> list)` and `convert(Iterable<TspType> list)` is not possible, because they still have the same erasure. 

One alternative would be name them differently, e.g., `convertString(Iterable<String> list)` and `convertTspType(Iterable<TspType> list)`.

Reply:
Ah, I did not realize that. Yeah, different names could be one option. I am still thinking we could avoid that though.